core/trino-main/src/main/java/io/trino/execution/DistributionSnapshot.java

```diff
@@ -110,15 +110,15 @@ public DistributionSnapshot(Distribution<?> distribution)
                 distribution.getTotal(),
```

Comment:
Can you add a test that exercises this method?

Reply:
+1